addOns/ascanrulesAlpha/src/main/java/org/zaproxy/zap/extension/ascanrulesAlpha/AbstractAppFilePlugin.java

```diff
@@ -168,6 +168,9 @@ public void scan() {
             return;
         }
         int statusCode = newRequest.getResponseHeader().getStatusCode();
+        if (isFalsePositive(newRequest)) {
```

Comment:
This could be added before the above statement.

Reply:
Fixed :)